beacon_node/beacon_chain/src/attestation_rewards.rs

```diff
@@ -152,6 +220,9 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
                 head: head_reward,
                 target: target_reward,
                 source: source_reward,
+                inclusion_delay: None,
+                // TODO: altair calculation logic needs to be updated to include inactivity penalty
```

Comment:
do you mind opening an issue for this? :pray:

Reply:
Done! https://github.com/sigp/lighthouse/issues/4512